shell/platform/darwin/macos/framework/Source/FlutterEngine.mm

```diff
@@ -309,14 +312,15 @@ - (BOOL)running {
 - (NSOpenGLContext*)resourceContext {
   if (!_resourceContext) {
     NSOpenGLPixelFormatAttribute attributes[] = {
-        NSOpenGLPFAColorSize, 24, NSOpenGLPFAAlphaSize, 8, NSOpenGLPFADoubleBuffer, 0,
```

Comment:
Why was `NSOpenGLPFADoubleBuffer` removed?

Reply:
Because it's no longer required. Instead of one double buffered NSOpenGLContexts, we'll have two single buffered NSOpenGLContexts each backed by different IOSurface.